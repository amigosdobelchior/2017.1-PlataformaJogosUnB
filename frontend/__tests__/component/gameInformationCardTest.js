import React from 'react';
import {Card, Image} from "semantic-ui-react";
import renderer from 'react-test-renderer';


test('Test render GameInformationCard', () => {
  const component = renderer.create(
      <Card fluid>
          <Image src="./image.jpg"/>

          <Card.Content>

            <Card.Description>
                  {"1.0"},
                  {"2016"},
                  {"nome"}
              </Card.Description>
          </Card.Content>

          <Card.Content extra>
              {"https://google.com"}
          </Card.Content>
      </Card>
	);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
