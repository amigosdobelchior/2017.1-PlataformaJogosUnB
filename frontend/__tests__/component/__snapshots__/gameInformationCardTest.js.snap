// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test render GameInformationCard 1`] = `
<div
  className="ui fluid card"
  href={undefined}
  onClick={[Function]}
>
  <img
    alt={undefined}
    className="ui image"
    height={undefined}
    src="./image.jpg"
    width={undefined}
  />
  <div
    className="content"
  >
    <div
      className="description"
    >
      1.0
      ,
      2016
      ,
      nome
    </div>
  </div>
  <div
    className="extra content"
  >
    https://google.com
  </div>
</div>
`;
